### PR TITLE
Validate that datetimes, dates, and times are valid

### PR DESCRIPTION
**Pull request by bryanculbertson**
Validate the fields with datetimes, dates, or times are valid isoformatted representations

### CODE REVIEW

File: vaccine_feed_ingest_schema/location.py [hunk 1]
```diff
@@ -221,22 +255,46 @@ class OpenDate(BaseModel):
...
+
+    @root_validator
+    @classmethod
+    def validate_closes_after_opens(cls, values: dict) -> dict:
+        opens = values.get("opens")
+        closes = values.get("closes")
+
+        if opens and closes:
+            if closes < opens:
+                raise ValueError("Closes date must be after opens date")
```
Comment (eliblock) [inline vaccine_feed_ingest_schema/location.py:269]:
```suggestion
                raise ValueError("Closes date must be on or after opens date")
```

File: vaccine_feed_ingest_schema/location.py [hunk 2]
```diff
@@ -221,22 +255,46 @@ class OpenDate(BaseModel):
...
+        "opens": str as 24h hh:mm,
+        "closes": str as 24h hh:mm,
     }
     """
 
     day: DayOfWeek
-    opens: str
-    closes: str
+    opens: StringTime
+    closes: StringTime
```
Comment (eliblock) [inline vaccine_feed_ingest_schema/location.py:285]: To confirm: we plan to compute the timezone based on the location on-display, so these should be in local time, right?

**Review by eliblock — APPROVED** [review]

File: vaccine_feed_ingest_schema/location.py (hunk 2 above)
Comment (bryanculbertson) [reply vaccine_feed_ingest_schema/location.py:285]: Yes, these should be in local time.

Comment (bryanculbertson) [reply vaccine_feed_ingest_schema/location.py:285]: We can add a timezone field if we need to later

**Pull request merged**